conftest.py

```
@@ -0,0 +1,284 @@
+        if config.option.keyword:
+            params.append(f"k={config.option.keyword}")
+        if config.option.markexpr:
+            params.append(f"m={config.option.markexpr}")
+
+        params_str = f"-{'-'.join(params)}" if params else ""
+        os.environ["EXPERIMENT_ID"] = (
+            f"{username}-{git_branch}{params_str}-{readable_timestamp()}"
+        )
+
+
+def pytest_report_header(config):
```

Ruff (0.14.14)

[warning] Unused function argument: `config`

(ARG001)